### DISCUSSION TITLE
No module named 'imp' Error during Run

### DISCUSSION THREAD

**Discussion by carldister** [labs4capella/python4capella#226] [Q&A]
Hello - I'm new to Python4Capella and followed the installation and configuration procedure carefully.  However< I am still getting and error when trying to run the Python4Capella scripts - Any ideas?

Capella Version: 7.0.0 Build id: 202407091438
 EASE Py4J Support (Incubation)	0.8.0.I202104091518
 Python4Capella	1.3.0.202408221204
 PyDev for Eclipse	9.3.0.202203051235

C:\Capella\capella\plugins\org.eclipse.ease.lang.python.py4j_0.8.0.I202104091518\pysrc\ease_py4j_main.py:414: DeprecationWarning: setDaemon() is deprecated, set the daemon attribute instead
  thread.setDaemon(True)
org.eclipse.ease.ScriptExecutionException: Traceback (most recent call last):
  File "<...>", line 17, in <module>
ModuleNotFoundError: No module named 'imp'

**Comment by ylussaud** [comment]
Python4Capella is built on top of EASE and one script from the version of EASE we are using is not compatible with Python 3.12. You need to use an older version of Python (see issue #216).

For the first warning you can have a look at issue #112.
